templates/takeovers/_edge-month_takeover.html

```diff
@@ -0,0 +1,34 @@
+<section class="p-strip--dark p-takeover js-takeover">
+  <div class="row u-clearfix u-vertically-center">
+    <div class="col-8">
+      <h1>
+        Canonical presents:<br/>Edge month
+      </h1>
+      <h4 class="p-heading--four u-sv3">
+        Join Canonical Product Managers for a month of webinars on edge
+        computing
```

Comment:
It matches the copy doc, but I think there needs to be a full stop (`.`) at the end of this sentence, to keep it consistent with other takeovers.